binancial/strategies/weighted_random.py

```
@@ -0,0 +1,32 @@
+class WeightedRandom:
+
+    def __init__(self, buy_weight=1, sell_weight=1, hold_weight=1):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)